api/jobs/queue.py

```diff
@@ -72,7 +72,6 @@ def mutate(job, mutation):
         if result.modified_count != 1:
             raise Exception('Job modification not saved')
 
-        # If the job did not succeed, check to see if job should be retried.
```

Comment:
That was probably removed by mistake, I'll that back in, thanks for catching it!